lib/smile-identity-core/signature.rb

```diff
@@ -6,14 +6,14 @@ def initialize(partner_id, api_key)
       @partner_id = partner_id
     end
 
+    # @deprecated since SDK version 2.0.0
```

Comment:
I agree here. we can remove the `{generate,confirm}_sec_key` methods.

or do we have a reason for keeping them @ArlonTM?
considering this is a major version bump i don't think we do.

Reply:
@tamssokari  I totally agree with both of you. 
It's just common practice (at least, in my opinion) to keep deprecated methods in the next release, just marking them as obsolete. It shows that we're gracefully phasing out a feature; otherwise it gives the impression that something's broken or just disappeared into thin air.
Anyway, this argument is moot now; I've removed said methods. 🙏🏽